lib/github_changelog_generator/octo_fetcher.rb

```diff
@@ -164,6 +164,7 @@ def fetch_closed_pull_requests
       pull_requests = []
       options = { state: "closed" }
 
+      # XXX Why does this override the base (HISTORY.md) setting?
```

Comment:
This is actually the `options` hash to pass to the `@client` not the `@options` hash for gcg.